src/PrestaShopBundle/Resources/config/services/core/grid/grid_definition_factory.yml

```diff
@@ -12,59 +11,59 @@ services:
 
   prestashop.core.grid.definition.factory.logs:
     class: 'PrestaShop\PrestaShop\Core\Grid\Definition\Factory\LogGridDefinitionFactory'
-    parent: 'prestashop.core.grid.definition.factory.abstract_grid_definition'
+    parent: PrestaShop\PrestaShop\Core\Grid\Definition\Factory\AbstractGridDefinitionFactory
```

Comment:
I like the idea to replace usages by fqcns, but would not it be better to just get rid of all the `parent` mention ? As they are adding complexing without any added value. This also stop us to use `defaults` as defaults and parent are not compatible

Reply:
yeah, but im pretty sure it needs dedicated PR. It would require too many global changes

Reply:
I've started doing it and it seems to require way too many changes, So if it is ok, Im not doing it in same PR. It is not some critical issue, but only an improvement not related to the PR, so it that can be (and I think must be) done in dedicated PR.

Reply:
Up to you, if it's too much work, just leave the old definition as it is, and we'll make this changes in a dedicated pr